packages/cli/src/test.ts

```diff
@@ -0,0 +1,74 @@
+import { parsePlatform } from "./parsers";
+
+type Args = {
+  platform: "android" | "ios" | "macos" | "windows" | "win32";
+};
+
+type ConfigT = Record<string, unknown>;
```

Comment:
The complete type is available. Even if you don't need it now, others may in the future.

```suggestion
import type { Config as CLIConfig } from "@react-native-community/cli-types";
```